pages/common/csvclean.md

```diff
@@ -0,0 +1,12 @@
+# csvclean
+
+> Finds and cleans common syntax errors in CSV files.
+> Included in csvkit.
```

Comment:
This may be misread as "including [errors] in csvkit." by an inattentive reader (e.g. me). How about making it explicit, i.e. "This command is included in csvkit"? Or is that overkill?